docs/_pages/0260-changelog.md

```diff
@@ -38,7 +39,7 @@ next_url: /docs/releases/
 
 ### Added
 
-* Added expiration and “invalid before” dates to secrets. 
+* Added expiration and “invalid before” dates to secrets.
```

Comment:
most of these are autoformatting of Zed editor.

I disabled “format on save”. It won’t happen next time.